pom.xml

```diff
@@ -465,6 +466,13 @@
                 <classifier>all</classifier>
             </dependency>
 
+            <dependency>
+                <groupId>com.databend</groupId>
+                <artifactId>databend-jdbc</artifactId>
+                <version>0.2.8</version>
```

Comment:
Move this dependency to `trino-databen` module.

Reply:
But why the other data source dependency here such as clickhouse, mysql 🤔